2018-03-12 21:45:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE NAME (
  ID INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
  RU NVARCHAR(150) NULL,
  KZ NVARCHAR(150) NULL,
  EN NVARCHAR(150)  NULL,
  RUGENITIVE NVARCHAR(150)  NULL,
  RUDATIVE NVARCHAR(150)  NULL,
  KZGENITIVE NVARCHAR(150)  NULL,
  KZDATIVE NVARCHAR(150) NULL,
  ENGENITIVE NVARCHAR(150)  NULL,
  ENDATIVE NVARCHAR(150)  NULL,
  RUABBR NVARCHAR(32) NULL,
  ENABBR NVARCHAR(32) NULL,
  KZABBR NVARCHAR(32) NULL)

CREATE[*] TABLE ACADEMICDEGREE (
  ID INT NOT NULL,
  NAMEID INT NOT NULL
)

CREATE TABLE ACADEMICTITLE (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL
)

CREATE TABLE EDUCATIONLEVEL (
  ID INT AUTO_INCREMENT NOT NULL,
  CODE VARCHAR(32) NULL,
  NAMEID INT NOT NULL,
  ISSPECIALITY BIT DEFAULT 0 NOT NULL,
  LEVEL INT  DEFAULT 0 NOT NULL,
  YEARBASE INT  DEFAULT 0 NOT NULL,
  ISCREDIT BIT  DEFAULT 0 NOT NULL
)

CREATE TABLE QUALIFICATION (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL,
  EDUCATIONLEVELID INT NULL
)

CREATE TABLE PLACEHIERARHY (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL,
  ISCODE BIT  DEFAULT 0 NOT NULL,
  ISSETTLEMENT BIT  DEFAULT 0 NOT NULL,
  POSITION INT  DEFAULT 0 NOT NULL,
  ISSEARCH BIT  DEFAULT 0 NOT NULL
)

CREATE TABLE PLACETYPE (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL,
  PLACEHIERARHYID INT NOT NULL,
  COUNT INT  DEFAULT 0 NOT NULL
)

CREATE TABLE PLACE (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL,
  PLACETYPEID INT NOT NULL,
  PARENTID INT NULL,
  PHONECODE VARCHAR(32) NULL,
  PHONEMASK VARCHAR(32) NULL,
  KATOCODE NVARCHAR(12) NULL
)

CREATE TABLE PHOTO (
  ID INT AUTO_INCREMENT NOT NULL,
  BYTES IMAGE NOT NULL,
  PERSONID INT NULL
)

CREATE TABLE NATIONALITY (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL
)

CREATE TABLE MIDDLENAME (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL
)

CREATE TABLE MARITALSTATUS (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL
)

CREATE TABLE LASTNAME (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL
)


CREATE TABLE GENDER (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL
)

CREATE TABLE KINSHIP (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT DEFAULT 10 NOT NULL,
  GENDERID INT NULL
)

CREATE TABLE FIRSTNAME (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL
)

CREATE TABLE DOCUMENTKIND (
  ID INT AUTO_INCREMENT NOT NULL,
  SERIESMASK VARCHAR(32) NULL,
  NUMBERMASK VARCHAR(32) NULL,
  NAMEID INT NOT NULL,
  ISSERIES BIT DEFAULT 1 NOT NULL,
  ISEDUCATIONCERTIFICATE BIT DEFAULT 1 NOT NULL,
  ISIIN BIT  DEFAULT 1 NOT NULL
)

CREATE TABLE DISABILITYGROUP (
  ID INT AUTO_INCREMENT NOT NULL,
  NAME VARCHAR(32) NOT NULL
)

CREATE TABLE BLOODGROUP (
  ID INT AUTO_INCREMENT NOT NULL,
  NAME VARCHAR(32) NOT NULL
)

CREATE TABLE ORGANIZATION (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL,
  PLACEID INT NULL,
  ISEDUCATIONAL BIT DEFAULT 0 NOT NULL,
  ISIDENTITYDOCUMENT BIT  DEFAULT 0 NOT NULL,
  ISMILITARY BIT  DEFAULT 0 NOT NULL
)

CREATE TABLE PERSON (
  ID INT AUTO_INCREMENT NOT NULL,
  LASTNAME NVARCHAR(32) NULL,
  FIRSTNAME NVARCHAR(32) NULL,
  MIDDLENAME NVARCHAR(32) NULL,
  IIN VARCHAR(12) NULL,
  RNN VARCHAR(12) NULL,
  BIRTHDATE DATE NULL,
  NATIONALITYID INT NULL,
  GENDERID INT NULL,
  MARITALSTATUSID INT NULL,
  BLOODGROUPID INT NULL,
  CITIZENSHIPPLACEID INT NULL,
  DISABILITYGROUPID INT NULL,
  ISPENSIONER BIT NULL,
  ISWORKLESS BIT NULL,
  ACADEMICTITLEID INT NULL,
  KINSHIPID INT NULL,
  SIC VARCHAR(16) NULL,
  BIRTHPLACEID INT NULL,
  REGISTRATIONPLACEID INT NULL,
  LIVEPLACEID INT NULL,
  DOCUMENTKINDID INT NULL,
  DOCUMENTSERIES VARCHAR(32) NULL,
  DOCUMENTNUMBER VARCHAR(32) NULL,
  DOCUMENTDATE DATE NULL,
  DOCUMENTORGANIZATIONID INT NULL,
  CANCHANGE BIT  DEFAULT 1 NOT NULL,
  QUALIFICATIONID INT NULL,
  PHOTOID INT NULL,
  LASTNAMEID INT NULL,
  FIRSTNAMEID INT NULL,
  MIDDLENAMEID INT NULL,
  EMAIL NVARCHAR(MAX) NULL
)

CREATE TABLE DEPARTMENT (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL
)

CREATE TABLE ORGANIZATIONSTRUCTURE (
  ID INT AUTO_INCREMENT NOT NULL,
  ORGANIZATIONID INT NOT NULL,
  DEPARTMENTID INT NOT NULL,
  PARENTID INT NULL,
  ISFILIAL BIT DEFAULT 0 NOT NULL,
  ISCHAIR BIT  DEFAULT 0 NOT NULL,
  ISFACULTY BIT  DEFAULT 0 NOT NULL
)

CREATE TABLE AUTHENTICATION (
  ID INT AUTO_INCREMENT NOT NULL,
  PERSONID INT NOT NULL,
  LOGIN NVARCHAR(32) NOT NULL,
  PASSWORD NVARCHAR(32) NOT NULL,
  DATETO DATETIME NULL,
  ORGANIZATIONID INT NULL,
  ORGANIZATIONSTRUCTUREID INT NULL,
  ISAUTHORIZED BIT  DEFAULT 0 NOT NULL
)

CREATE TABLE MODULE (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NULL,
  FORMNAME VARCHAR(64) NULL,
  ICONPATH VARCHAR(128) NULL,
  PARENTID INT NULL,
  POSITION INT DEFAULT 0 NOT NULL,
  SETTINGS NVARCHAR(1024) NULL
) "
Syntax error in SQL statement "CREATE TABLE NAME (
  ID INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
  RU NVARCHAR(150) NULL,
  KZ NVARCHAR(150) NULL,
  EN NVARCHAR(150)  NULL,
  RUGENITIVE NVARCHAR(150)  NULL,
  RUDATIVE NVARCHAR(150)  NULL,
  KZGENITIVE NVARCHAR(150)  NULL,
  KZDATIVE NVARCHAR(150) NULL,
  ENGENITIVE NVARCHAR(150)  NULL,
  ENDATIVE NVARCHAR(150)  NULL,
  RUABBR NVARCHAR(32) NULL,
  ENABBR NVARCHAR(32) NULL,
  KZABBR NVARCHAR(32) NULL)

CREATE[*] TABLE ACADEMICDEGREE (
  ID INT NOT NULL,
  NAMEID INT NOT NULL
)

CREATE TABLE ACADEMICTITLE (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL
)

CREATE TABLE EDUCATIONLEVEL (
  ID INT AUTO_INCREMENT NOT NULL,
  CODE VARCHAR(32) NULL,
  NAMEID INT NOT NULL,
  ISSPECIALITY BIT DEFAULT 0 NOT NULL,
  LEVEL INT  DEFAULT 0 NOT NULL,
  YEARBASE INT  DEFAULT 0 NOT NULL,
  ISCREDIT BIT  DEFAULT 0 NOT NULL
)

CREATE TABLE QUALIFICATION (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL,
  EDUCATIONLEVELID INT NULL
)

CREATE TABLE PLACEHIERARHY (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL,
  ISCODE BIT  DEFAULT 0 NOT NULL,
  ISSETTLEMENT BIT  DEFAULT 0 NOT NULL,
  POSITION INT  DEFAULT 0 NOT NULL,
  ISSEARCH BIT  DEFAULT 0 NOT NULL
)

CREATE TABLE PLACETYPE (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL,
  PLACEHIERARHYID INT NOT NULL,
  COUNT INT  DEFAULT 0 NOT NULL
)

CREATE TABLE PLACE (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL,
  PLACETYPEID INT NOT NULL,
  PARENTID INT NULL,
  PHONECODE VARCHAR(32) NULL,
  PHONEMASK VARCHAR(32) NULL,
  KATOCODE NVARCHAR(12) NULL
)

CREATE TABLE PHOTO (
  ID INT AUTO_INCREMENT NOT NULL,
  BYTES IMAGE NOT NULL,
  PERSONID INT NULL
)

CREATE TABLE NATIONALITY (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL
)

CREATE TABLE MIDDLENAME (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL
)

CREATE TABLE MARITALSTATUS (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL
)

CREATE TABLE LASTNAME (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL
)


CREATE TABLE GENDER (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL
)

CREATE TABLE KINSHIP (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT DEFAULT 10 NOT NULL,
  GENDERID INT NULL
)

CREATE TABLE FIRSTNAME (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL
)

CREATE TABLE DOCUMENTKIND (
  ID INT AUTO_INCREMENT NOT NULL,
  SERIESMASK VARCHAR(32) NULL,
  NUMBERMASK VARCHAR(32) NULL,
  NAMEID INT NOT NULL,
  ISSERIES BIT DEFAULT 1 NOT NULL,
  ISEDUCATIONCERTIFICATE BIT DEFAULT 1 NOT NULL,
  ISIIN BIT  DEFAULT 1 NOT NULL
)

CREATE TABLE DISABILITYGROUP (
  ID INT AUTO_INCREMENT NOT NULL,
  NAME VARCHAR(32) NOT NULL
)

CREATE TABLE BLOODGROUP (
  ID INT AUTO_INCREMENT NOT NULL,
  NAME VARCHAR(32) NOT NULL
)

CREATE TABLE ORGANIZATION (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL,
  PLACEID INT NULL,
  ISEDUCATIONAL BIT DEFAULT 0 NOT NULL,
  ISIDENTITYDOCUMENT BIT  DEFAULT 0 NOT NULL,
  ISMILITARY BIT  DEFAULT 0 NOT NULL
)

CREATE TABLE PERSON (
  ID INT AUTO_INCREMENT NOT NULL,
  LASTNAME NVARCHAR(32) NULL,
  FIRSTNAME NVARCHAR(32) NULL,
  MIDDLENAME NVARCHAR(32) NULL,
  IIN VARCHAR(12) NULL,
  RNN VARCHAR(12) NULL,
  BIRTHDATE DATE NULL,
  NATIONALITYID INT NULL,
  GENDERID INT NULL,
  MARITALSTATUSID INT NULL,
  BLOODGROUPID INT NULL,
  CITIZENSHIPPLACEID INT NULL,
  DISABILITYGROUPID INT NULL,
  ISPENSIONER BIT NULL,
  ISWORKLESS BIT NULL,
  ACADEMICTITLEID INT NULL,
  KINSHIPID INT NULL,
  SIC VARCHAR(16) NULL,
  BIRTHPLACEID INT NULL,
  REGISTRATIONPLACEID INT NULL,
  LIVEPLACEID INT NULL,
  DOCUMENTKINDID INT NULL,
  DOCUMENTSERIES VARCHAR(32) NULL,
  DOCUMENTNUMBER VARCHAR(32) NULL,
  DOCUMENTDATE DATE NULL,
  DOCUMENTORGANIZATIONID INT NULL,
  CANCHANGE BIT  DEFAULT 1 NOT NULL,
  QUALIFICATIONID INT NULL,
  PHOTOID INT NULL,
  LASTNAMEID INT NULL,
  FIRSTNAMEID INT NULL,
  MIDDLENAMEID INT NULL,
  EMAIL NVARCHAR(MAX) NULL
)

CREATE TABLE DEPARTMENT (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NOT NULL
)

CREATE TABLE ORGANIZATIONSTRUCTURE (
  ID INT AUTO_INCREMENT NOT NULL,
  ORGANIZATIONID INT NOT NULL,
  DEPARTMENTID INT NOT NULL,
  PARENTID INT NULL,
  ISFILIAL BIT DEFAULT 0 NOT NULL,
  ISCHAIR BIT  DEFAULT 0 NOT NULL,
  ISFACULTY BIT  DEFAULT 0 NOT NULL
)

CREATE TABLE AUTHENTICATION (
  ID INT AUTO_INCREMENT NOT NULL,
  PERSONID INT NOT NULL,
  LOGIN NVARCHAR(32) NOT NULL,
  PASSWORD NVARCHAR(32) NOT NULL,
  DATETO DATETIME NULL,
  ORGANIZATIONID INT NULL,
  ORGANIZATIONSTRUCTUREID INT NULL,
  ISAUTHORIZED BIT  DEFAULT 0 NOT NULL
)

CREATE TABLE MODULE (
  ID INT AUTO_INCREMENT NOT NULL,
  NAMEID INT NULL,
  FORMNAME VARCHAR(64) NULL,
  ICONPATH VARCHAR(128) NULL,
  PARENTID INT NULL,
  POSITION INT DEFAULT 0 NOT NULL,
  SETTINGS NVARCHAR(1024) NULL
) "; SQL statement:
CREATE TABLE Name (
  Id int PRIMARY KEY auto_increment NOT NULL,
  Ru nvarchar(150) NULL,
  Kz nvarchar(150) NULL,
  En nvarchar(150)  NULL,
  RuGenitive nvarchar(150)  NULL,
  RuDative nvarchar(150)  NULL,
  KzGenitive nvarchar(150)  NULL,
  KzDative nvarchar(150) NULL,
  EnGenitive nvarchar(150)  NULL,
  EnDative nvarchar(150)  NULL,
  RuAbbr nvarchar(32) NULL,
  EnAbbr nvarchar(32) NULL,
  KzAbbr nvarchar(32) NULL)

CREATE TABLE AcademicDegree (
  Id int NOT NULL,
  NameId int NOT NULL
)

CREATE TABLE AcademicTitle (
  Id int AUTO_INCREMENT NOT NULL,
  NameId int NOT NULL
)

CREATE TABLE EducationLevel (
  Id int AUTO_INCREMENT NOT NULL,
  Code varchar(32) NULL,
  NameId int NOT NULL,
  IsSpeciality bit DEFAULT 0 NOT NULL,
  Level int  DEFAULT 0 NOT NULL,
  YearBase int  DEFAULT 0 NOT NULL,
  IsCredit bit  DEFAULT 0 NOT NULL
)

CREATE TABLE Qualification (
  Id int AUTO_INCREMENT NOT NULL,
  NameId int NOT NULL,
  EducationLevelId int NULL
)

CREATE TABLE PlaceHierarhy (
  Id int AUTO_INCREMENT NOT NULL,
  NameId int NOT NULL,
  IsCode bit  DEFAULT 0 NOT NULL,
  IsSettlement bit  DEFAULT 0 NOT NULL,
  Position int  DEFAULT 0 NOT NULL,
  IsSearch bit  DEFAULT 0 NOT NULL
)

CREATE TABLE PlaceType (
  Id int AUTO_INCREMENT NOT NULL,
  NameId int NOT NULL,
  PlaceHierarhyId int NOT NULL,
  Count int  DEFAULT 0 NOT NULL
)

CREATE TABLE Place (
  Id int AUTO_INCREMENT NOT NULL,
  NameId int NOT NULL,
  PlaceTypeId int NOT NULL,
  ParentId int NULL,
  PhoneCode varchar(32) NULL,
  PhoneMask varchar(32) NULL,
  KatoCode nvarchar(12) NULL
)

CREATE TABLE Photo (
  Id int AUTO_INCREMENT NOT NULL,
  Bytes image NOT NULL,
  PersonId int NULL
)

CREATE TABLE Nationality (
  Id int AUTO_INCREMENT NOT NULL,
  NameId int NOT NULL
)

CREATE TABLE MiddleName (
  Id int AUTO_INCREMENT NOT NULL,
  NameId int NOT NULL
)

CREATE TABLE MaritalStatus (
  Id int AUTO_INCREMENT NOT NULL,
  NameId int NOT NULL
)

CREATE TABLE LastName (
  Id int AUTO_INCREMENT NOT NULL,
  NameId int NOT NULL
)


CREATE TABLE Gender (
  Id int AUTO_INCREMENT NOT NULL,
  NameId int NOT NULL
)

CREATE TABLE Kinship (
  Id int AUTO_INCREMENT NOT NULL,
  NameId int DEFAULT 10 NOT NULL,
  GenderId int NULL
)

CREATE TABLE FirstName (
  Id int AUTO_INCREMENT NOT NULL,
  NameId int NOT NULL
)

CREATE TABLE DocumentKind (
  Id int AUTO_INCREMENT NOT NULL,
  SeriesMask varchar(32) NULL,
  NumberMask varchar(32) NULL,
  NameId int NOT NULL,
  IsSeries bit DEFAULT 1 NOT NULL,
  IsEducationCertificate bit DEFAULT 1 NOT NULL,
  IsIin bit  DEFAULT 1 NOT NULL
)

CREATE TABLE DisabilityGroup (
  Id int AUTO_INCREMENT NOT NULL,
  Name varchar(32) NOT NULL
)

CREATE TABLE BloodGroup (
  Id int AUTO_INCREMENT NOT NULL,
  Name varchar(32) NOT NULL
)

CREATE TABLE Organization (
  Id int AUTO_INCREMENT NOT NULL,
  NameId int NOT NULL,
  PlaceId int NULL,
  IsEducational bit DEFAULT 0 NOT NULL,
  IsIdentityDocument bit  DEFAULT 0 NOT NULL,
  IsMilitary bit  DEFAULT 0 NOT NULL
)

CREATE TABLE Person (
  Id int AUTO_INCREMENT NOT NULL,
  LastName nvarchar(32) NULL,
  FirstName nvarchar(32) NULL,
  MiddleName nvarchar(32) NULL,
  Iin varchar(12) NULL,
  Rnn varchar(12) NULL,
  BirthDate date NULL,
  NationalityId int NULL,
  GenderId int NULL,
  MaritalStatusId int NULL,
  BloodGroupId int NULL,
  CitizenshipPlaceId int NULL,
  DisabilityGroupId int NULL,
  IsPensioner bit NULL,
  IsWorkless bit NULL,
  AcademicTitleId int NULL,
  KinshipId int NULL,
  Sic varchar(16) NULL,
  BirthPlaceId int NULL,
  RegistrationPlaceId int NULL,
  LivePlaceId int NULL,
  DocumentKindId int NULL,
  DocumentSeries varchar(32) NULL,
  DocumentNumber varchar(32) NULL,
  DocumentDate date NULL,
  DocumentOrganizationId int NULL,
  CanChange bit  DEFAULT 1 NOT NULL,
  QualificationId int NULL,
  PhotoId int NULL,
  LastNameId int NULL,
  FirstNameId int NULL,
  MiddleNameId int NULL,
  Email nvarchar(max) NULL
)

CREATE TABLE Department (
  Id int AUTO_INCREMENT NOT NULL,
  NameId int NOT NULL
)

CREATE TABLE OrganizationStructure (
  Id int AUTO_INCREMENT NOT NULL,
  OrganizationId int NOT NULL,
  DepartmentId int NOT NULL,
  ParentId int NULL,
  IsFilial bit DEFAULT 0 NOT NULL,
  IsChair bit  DEFAULT 0 NOT NULL,
  IsFaculty bit  DEFAULT 0 NOT NULL
)

CREATE TABLE Authentication (
  Id int AUTO_INCREMENT NOT NULL,
  PersonId int NOT NULL,
  Login nvarchar(32) NOT NULL,
  Password nvarchar(32) NOT NULL,
  DateTo datetime NULL,
  OrganizationId int NULL,
  OrganizationStructureId int NULL,
  IsAuthorized bit  DEFAULT 0 NOT NULL
)

CREATE TABLE Module (
  Id int AUTO_INCREMENT NOT NULL,
  NameId int NULL,
  FormName varchar(64) NULL,
  IconPath varchar(128) NULL,
  ParentId int NULL,
  Position int DEFAULT 0 NOT NULL,
  Settings nvarchar(1024) NULL
) [42000-196]
2018-03-12 21:46:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "KINSHIP" уже существует
Table "KINSHIP" already exists; SQL statement:
CREATE TABLE Kinship (
  Id int AUTO_INCREMENT NOT NULL,
  NameId int DEFAULT 10 NOT NULL,
  GenderId int NULL
) [42101-196]
2018-03-12 21:49:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE BUILDINGSTRUCTURE (
  ID INT AUTO_INCREMENT NOT FOR[*] REPLICATION NOT NULL,
  ORGANIZATIONID INT NOT NULL,
  NAMEID INT NOT NULL,
  PLACEID INT NULL,
  PARENTID INT NULL,
  PLACECOUNT INT  DEFAULT 0 NOT NULL,
  ISLIVING BIT  DEFAULT 0 NOT NULL,
  ISEDUCATION BIT DEFAULT 0 NOT NULL,
  FILIALID INT NOT NULL
) "; ожидалось "NULL"
Syntax error in SQL statement "CREATE TABLE BUILDINGSTRUCTURE (
  ID INT AUTO_INCREMENT NOT FOR[*] REPLICATION NOT NULL,
  ORGANIZATIONID INT NOT NULL,
  NAMEID INT NOT NULL,
  PLACEID INT NULL,
  PARENTID INT NULL,
  PLACECOUNT INT  DEFAULT 0 NOT NULL,
  ISLIVING BIT  DEFAULT 0 NOT NULL,
  ISEDUCATION BIT DEFAULT 0 NOT NULL,
  FILIALID INT NOT NULL
) "; expected "NULL"; SQL statement:
CREATE TABLE BuildingStructure (
  Id int AUTO_INCREMENT NOT FOR REPLICATION NOT NULL,
  OrganizationId int NOT NULL,
  NameId int NOT NULL,
  PlaceId int NULL,
  ParentId int NULL,
  PlaceCount int  DEFAULT 0 NOT NULL,
  IsLiving bit  DEFAULT 0 NOT NULL,
  IsEducation bit DEFAULT 0 NOT NULL,
  FilialId int NOT NULL
) [42001-196]
2018-03-12 21:50:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Функция "SYSDATETIME" не найдена
Function "SYSDATETIME" not found; SQL statement:
CREATE TABLE ChangeLog (
  Id int AUTO_INCREMENT NOT NULL,
  DateTime datetime2(1)  DEFAULT sysdatetime() NOT NULL,
  PersonId int NOT NULL,
  Data nvarchar(2048) NULL,
  ChangeType varchar(1) NOT NULL,
  TableName varchar(128) NULL,
  String nvarchar(256) NULL,
  ItemId int DEFAULT 0 NOT NULL,
  OldData nvarchar(2048) NULL,
  IsCanceled bit  DEFAULT 0 NOT NULL
) [90022-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.readJavaFunction(Parser.java:2487)
	at org.h2.command.Parser.readFunction(Parser.java:2539)
	at org.h2.command.Parser.readTerm(Parser.java:2876)
	at org.h2.command.Parser.readFactor(Parser.java:2390)
	at org.h2.command.Parser.readSum(Parser.java:2377)
	at org.h2.command.Parser.readConcat(Parser.java:2347)
	at org.h2.command.Parser.readCondition(Parser.java:2178)
	at org.h2.command.Parser.readAnd(Parser.java:2150)
	at org.h2.command.Parser.readExpression(Parser.java:2142)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4035)
	at org.h2.command.Parser.parseCreateTable(Parser.java:6155)
	at org.h2.command.Parser.parseCreate(Parser.java:4351)
	at org.h2.command.Parser.parsePrepared(Parser.java:366)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-12 21:50:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Столбец "SYSDATETIME" не найден
Column "SYSDATETIME" not found; SQL statement:
CREATE TABLE ChangeLog (
  Id int AUTO_INCREMENT NOT NULL,
  DateTime datetime2(1)  DEFAULT sysdatetime NOT NULL,
  PersonId int NOT NULL,
  Data nvarchar(2048) NULL,
  ChangeType varchar(1) NOT NULL,
  TableName varchar(128) NULL,
  String nvarchar(256) NULL,
  ItemId int DEFAULT 0 NOT NULL,
  OldData nvarchar(2048) NULL,
  IsCanceled bit  DEFAULT 0 NOT NULL
) [42122-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.table.Column.setDefaultExpression(Column.java:233)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4036)
	at org.h2.command.Parser.parseCreateTable(Parser.java:6155)
	at org.h2.command.Parser.parseCreate(Parser.java:4351)
	at org.h2.command.Parser.parsePrepared(Parser.java:366)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-12 21:54:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE ORDER[*] (
  ID INT AUTO_INCREMENT NOT NULL,
  NUMBER NVARCHAR(32) NOT NULL,
  DATE DATE NOT NULL,
  ORDERTYPEID INT NULL,
  COMMENT NVARCHAR(1024) NULL,
  ORGANIZATIONID INT NOT NULL
) "; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE ORDER[*] (
  ID INT AUTO_INCREMENT NOT NULL,
  NUMBER NVARCHAR(32) NOT NULL,
  DATE DATE NOT NULL,
  ORDERTYPEID INT NULL,
  COMMENT NVARCHAR(1024) NULL,
  ORGANIZATIONID INT NOT NULL
) "; expected "identifier"; SQL statement:
CREATE TABLE Order (
  Id int AUTO_INCREMENT NOT NULL,
  Number nvarchar(32) NOT NULL,
  Date date NOT NULL,
  OrderTypeId int NULL,
  Comment nvarchar(1024) NULL,
  OrganizationId int NOT NULL
) [42001-196]
2018-03-12 21:55:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE ORDER[*] (
  ID INT AUTO_INCREMENT NOT NULL,
  NUMBER NVARCHAR(32) NOT NULL,
  DATE DATE NOT NULL,
  ORDERTYPEID INT NULL,
  COMMENT NVARCHAR(1024) NULL,
  ORGANIZATIONID INT NOT NULL
) "; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE ORDER[*] (
  ID INT AUTO_INCREMENT NOT NULL,
  NUMBER NVARCHAR(32) NOT NULL,
  DATE DATE NOT NULL,
  ORDERTYPEID INT NULL,
  COMMENT NVARCHAR(1024) NULL,
  ORGANIZATIONID INT NOT NULL
) "; expected "identifier"; SQL statement:
CREATE TABLE Order (
  ID int AUTO_INCREMENT NOT NULL,
  Number nvarchar(32) NOT NULL,
  Date date NOT NULL,
  OrderTypeId int NULL,
  Comment nvarchar(1024) NULL,
  OrganizationId int NOT NULL
) [42001-196]
2018-03-12 21:59:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE GROUP[*] (
  ID INT AUTO_INCREMENT NOT NULL,
  EDUCATIONPROGRAMID INT NOT NULL,
  LANGUAGEID INT NOT NULL,
  ENTERYEAR INT NOT NULL,
  NUMBER INT NOT NULL,
  COMMENT VARCHAR(32) NULL,
  CURATORWORKID INT NULL,
  FILIALID INT NOT NULL,
  EDUCATIONCOUNT INT DEFAULT 0 NOT NULL,
  ADVISERWORKID INT NULL,
  ORGANIZATIONID INT NOT NULL,
  EDUCATIONSTATUSID INT NULL,
  CUSTOMNAME NVARCHAR(32) NULL,
  EDUCATIONDIRECTIONID INT NULL,
  EDUCATIONLEVELID INT NULL
) "; ожидалось "identifier"
Syntax error in SQL statement "CREATE TABLE GROUP[*] (
  ID INT AUTO_INCREMENT NOT NULL,
  EDUCATIONPROGRAMID INT NOT NULL,
  LANGUAGEID INT NOT NULL,
  ENTERYEAR INT NOT NULL,
  NUMBER INT NOT NULL,
  COMMENT VARCHAR(32) NULL,
  CURATORWORKID INT NULL,
  FILIALID INT NOT NULL,
  EDUCATIONCOUNT INT DEFAULT 0 NOT NULL,
  ADVISERWORKID INT NULL,
  ORGANIZATIONID INT NOT NULL,
  EDUCATIONSTATUSID INT NULL,
  CUSTOMNAME NVARCHAR(32) NULL,
  EDUCATIONDIRECTIONID INT NULL,
  EDUCATIONLEVELID INT NULL
) "; expected "identifier"; SQL statement:
CREATE TABLE Group (
  Id int AUTO_INCREMENT NOT NULL,
  EducationProgramId int NOT NULL,
  LanguageId int NOT NULL,
  EnterYear int NOT NULL,
  Number int NOT NULL,
  Comment varchar(32) NULL,
  CuratorWorkId int NULL,
  FilialId int NOT NULL,
  EducationCount int DEFAULT 0 NOT NULL,
  AdviserWorkId int NULL,
  OrganizationId int NOT NULL,
  EducationStatusId int NULL,
  CustomName nvarchar(32) NULL,
  EducationDirectionId int NULL,
  EducationLevelId int NULL
) [42001-196]
2018-03-13 10:10:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Неизвестный тип данных: "SYSNAME"
Unknown data type: "SYSNAME"; SQL statement:
CREATE TABLE sysdiagrams (
  name sysname NOT NULL,
  principal_id int NOT NULL,
  diagram_id int AUTO_INCREMENT NOT NULL,
  version int NULL,
  definition varbinary(max) NULL
) [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseCreateTable(Parser.java:6155)
	at org.h2.command.Parser.parseCreate(Parser.java:4351)
	at org.h2.command.Parser.parsePrepared(Parser.java:366)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 10:11:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "ALTER TABLE NAME
ADD CONSTRAINT PK__NAME__3214EC0715A53433 
PRIMARY KEY CLUSTERED[*] (ID)
WITH (
  PAD_INDEX = OFF,
  IGNORE_DUP_KEY = OFF,
  STATISTICS_NORECOMPUTE = OFF,
  ALLOW_ROW_LOCKS = ON,
  ALLOW_PAGE_LOCKS = ON)
ON PRIMARY
GO "; ожидалось "HASH, ("
Syntax error in SQL statement "ALTER TABLE NAME
ADD CONSTRAINT PK__NAME__3214EC0715A53433 
PRIMARY KEY CLUSTERED[*] (ID)
WITH (
  PAD_INDEX = OFF,
  IGNORE_DUP_KEY = OFF,
  STATISTICS_NORECOMPUTE = OFF,
  ALLOW_ROW_LOCKS = ON,
  ALLOW_PAGE_LOCKS = ON)
ON PRIMARY
GO "; expected "HASH, ("; SQL statement:
ALTER TABLE Name
ADD CONSTRAINT PK__Name__3214EC0715A53433 
PRIMARY KEY CLUSTERED (Id)
WITH (
  PAD_INDEX = OFF,
  IGNORE_DUP_KEY = OFF,
  STATISTICS_NORECOMPUTE = OFF,
  ALLOW_ROW_LOCKS = ON,
  ALLOW_PAGE_LOCKS = ON)
ON PRIMARY
GO [42001-196]
2018-03-13 10:36:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "ALTER TABLE NAME
ADD PRIMARY KEY (ID)
WITH[*] (
  PAD_INDEX = OFF,
  IGNORE_DUP_KEY = OFF,
  STATISTICS_NORECOMPUTE = OFF,
  ALLOW_ROW_LOCKS = ON,
  ALLOW_PAGE_LOCKS = ON)
ON PRIMARY
GO "
Syntax error in SQL statement "ALTER TABLE NAME
ADD PRIMARY KEY (ID)
WITH[*] (
  PAD_INDEX = OFF,
  IGNORE_DUP_KEY = OFF,
  STATISTICS_NORECOMPUTE = OFF,
  ALLOW_ROW_LOCKS = ON,
  ALLOW_PAGE_LOCKS = ON)
ON PRIMARY
GO "; SQL statement:
ALTER TABLE Name
ADD PRIMARY KEY (Id)
WITH (
  PAD_INDEX = OFF,
  IGNORE_DUP_KEY = OFF,
  STATISTICS_NORECOMPUTE = OFF,
  ALLOW_ROW_LOCKS = ON,
  ALLOW_PAGE_LOCKS = ON)
ON PRIMARY
GO [42000-196]
2018-03-13 10:37:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Попытка создания второго первичного ключа
Attempt to define a second primary key; SQL statement:
ALTER TABLE Name
ADD PRIMARY KEY (Id) [90017-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:124)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 10:47:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TABLE" не найдена
Table "TABLE" not found; SQL statement:
delete TABLE DisabilityGroup
ADD PRIMARY KEY (Id) [42102-196]
2018-03-13 10:47:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "DELETE DISABILITYGROUP
ADD PRIMARY[*] KEY (ID) "
Syntax error in SQL statement "DELETE DISABILITYGROUP
ADD PRIMARY[*] KEY (ID) "; SQL statement:
delete DisabilityGroup
ADD PRIMARY KEY (Id) [42000-196]
2018-03-13 10:48:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "DISABILITYGROUP" уже существует
Table "DISABILITYGROUP" already exists; SQL statement:
CREATE TABLE DisabilityGroup (
  Id int PRIMARY KEY AUTO_INCREMENT NOT NULL,
  Name varchar(32) NOT NULL UNIQUE 
) [42101-196]
2018-03-13 10:48:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "DISABILITYGROUP" уже существует
Table "DISABILITYGROUP" already exists; SQL statement:
CREATE TABLE DisabilityGroup (
  Id int PRIMARY KEY AUTO_INCREMENT NOT NULL,
  Name varchar(32) NOT NULL UNIQUE 
) [42101-196]
2018-03-13 10:51:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "ALTER TABLE DISABILITYGROUP 
ADD CONSTRAINT 
UNIQUE[*] NONCLUSTERED (NAME) "; ожидалось "identifier"
Syntax error in SQL statement "ALTER TABLE DISABILITYGROUP 
ADD CONSTRAINT 
UNIQUE[*] NONCLUSTERED (NAME) "; expected "identifier"; SQL statement:
alter table DisabilityGroup 
ADD CONSTRAINT 
UNIQUE NONCLUSTERED (Name) [42001-196]
2018-03-13 10:51:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "ALTER TABLE DISABILITYGROUP 
ADD CONSTRAINT 
UNIQUE[*]  (NAME) "; ожидалось "identifier"
Syntax error in SQL statement "ALTER TABLE DISABILITYGROUP 
ADD CONSTRAINT 
UNIQUE[*]  (NAME) "; expected "identifier"; SQL statement:
alter table DisabilityGroup 
ADD CONSTRAINT 
UNIQUE  (Name) [42001-196]
2018-03-13 10:54:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Неизвестный тип данных: "UNIQUE"
Unknown data type: "UNIQUE"; SQL statement:
ALTER TABLE BloodGroup
ADD identity 
UNIQUE NONCLUSTERED (Name) [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:5903)
	at org.h2.command.Parser.parseAlterTable(Parser.java:5615)
	at org.h2.command.Parser.parseAlter(Parser.java:5007)
	at org.h2.command.Parser.parsePrepared(Parser.java:348)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 10:54:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Неизвестный тип данных: "UNIQUE"
Unknown data type: "UNIQUE"; SQL statement:
ALTER TABLE BloodGroup
ADD identity 
UNIQUE  (Name) [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:5903)
	at org.h2.command.Parser.parseAlterTable(Parser.java:5615)
	at org.h2.command.Parser.parseAlter(Parser.java:5007)
	at org.h2.command.Parser.parsePrepared(Parser.java:348)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 10:56:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Попытка создания второго первичного ключа
Attempt to define a second primary key; SQL statement:
alter table Organization 
ADD PRIMARY KEY (Id) [90017-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:124)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 11:36:38 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "ALTER TABLE ACADEMICTITLE
ADD CONSTRAINT ACADEMICTITLE_FK_NAME FOREIGN KEY (NAMEID) 
  REFERENCES NAME (ID) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION
GO[*] "
Syntax error in SQL statement "ALTER TABLE ACADEMICTITLE
ADD CONSTRAINT ACADEMICTITLE_FK_NAME FOREIGN KEY (NAMEID) 
  REFERENCES NAME (ID) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION
GO[*] "; SQL statement:
ALTER TABLE AcademicTitle
ADD CONSTRAINT AcademicTitle_fk_Name FOREIGN KEY (NameId) 
  REFERENCES Name (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION
GO [42000-196]
2018-03-13 11:37:19 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "EDUCATIONLEVEL_FK_NAME" уже существует
Constraint "EDUCATIONLEVEL_FK_NAME" already exists; SQL statement:
ALTER TABLE EducationLevel
ADD CONSTRAINT EducationLevel_fk_Name FOREIGN KEY (NameId) 
  REFERENCES Name (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 11:40:05 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "ORGANIZATION_FK_NAME" уже существует
Constraint "ORGANIZATION_FK_NAME" already exists; SQL statement:
ALTER TABLE Organization
ADD CONSTRAINT Organization_fk_Name FOREIGN KEY (NameId) 
  REFERENCES Name (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 11:43:34 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "ALTER TABLE WORKSTATUSCHANGEREASON
ADD CONSTRAINT WORKSTATUSCHANGEREASON_FK_ NAME[*] FOREIGN KEY (NAMEID) 
  REFERENCES NAME (ID) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION "; ожидалось "., COMMENT, PRIMARY, INDEX, KEY, CHECK, UNIQUE, FOREIGN"
Syntax error in SQL statement "ALTER TABLE WORKSTATUSCHANGEREASON
ADD CONSTRAINT WORKSTATUSCHANGEREASON_FK_ NAME[*] FOREIGN KEY (NAMEID) 
  REFERENCES NAME (ID) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION "; expected "., COMMENT, PRIMARY, INDEX, KEY, CHECK, UNIQUE, FOREIGN"; SQL statement:
ALTER TABLE WorkStatusChangeReason
ADD CONSTRAINT WorkStatusChangeReason_fk_ Name FOREIGN KEY (NameId) 
  REFERENCES Name (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [42001-196]
2018-03-13 12:23:24 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "CURRICULUMMODELCYCLE_FK_NAME" уже существует
Constraint "CURRICULUMMODELCYCLE_FK_NAME" already exists; SQL statement:
ALTER TABLE CurriculumModelCycle
ADD CONSTRAINT CurriculumModelCycle_fk_Name FOREIGN KEY (NameId) 
  REFERENCES Name (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:23:31 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "CURRICULUMMODELCYCLE_FK_NAME" уже существует
Constraint "CURRICULUMMODELCYCLE_FK_NAME" already exists; SQL statement:
ALTER TABLE CurriculumModelCycle
ADD CONSTRAINT CurriculumModelCycle_fk_Name FOREIGN KEY (NameId) 
  REFERENCES Name (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:23:39 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "CURRICULUMMODELCYCLE_FK_NAME" уже существует
Constraint "CURRICULUMMODELCYCLE_FK_NAME" already exists; SQL statement:
ALTER TABLE CurriculumModelCycle
ADD CONSTRAINT CurriculumModelCycle_fk_Name FOREIGN KEY (NameId) 
  REFERENCES Name (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:24:33 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "DISCIPLINE_FK_EDUCATIONLEVEL" уже существует
Constraint "DISCIPLINE_FK_EDUCATIONLEVEL" already exists; SQL statement:
ALTER TABLE Discipline
ADD CONSTRAINT Discipline_fk_EducationLevel FOREIGN KEY (EducationLevelId) 
  REFERENCES EducationLevel (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:24:42 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "DISCIPLINE_FK_LANGUAGE" уже существует
Constraint "DISCIPLINE_FK_LANGUAGE" already exists; SQL statement:
ALTER TABLE Discipline
ADD CONSTRAINT Discipline_fk_Language FOREIGN KEY (LanguageId) 
  REFERENCES Language (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:24:49 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "DISCIPLINE_FK_ORGANIZATIONSTRUCTURE" уже существует
Constraint "DISCIPLINE_FK_ORGANIZATIONSTRUCTURE" already exists; SQL statement:
ALTER TABLE Discipline
ADD CONSTRAINT Discipline_fk_OrganizationStructure FOREIGN KEY (OrganizationStructureId) 
  REFERENCES OrganizationStructure (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:24:57 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "DISCIPLINE_FK_SUBJECT" уже существует
Constraint "DISCIPLINE_FK_SUBJECT" already exists; SQL statement:
ALTER TABLE Discipline
ADD CONSTRAINT Discipline_fk_Subject FOREIGN KEY (SubjectId) 
  REFERENCES Subject (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:25:04 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "DISCIPLINE_FK_EDUCATIONLEVEL" уже существует
Constraint "DISCIPLINE_FK_EDUCATIONLEVEL" already exists; SQL statement:
ALTER TABLE Discipline
ADD CONSTRAINT Discipline_fk_EducationLevel FOREIGN KEY (EducationLevelId) 
  REFERENCES EducationLevel (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:25:12 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "DISCIPLINE_FK_LANGUAGE" уже существует
Constraint "DISCIPLINE_FK_LANGUAGE" already exists; SQL statement:
ALTER TABLE Discipline
ADD CONSTRAINT Discipline_fk_Language FOREIGN KEY (LanguageId) 
  REFERENCES Language (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:25:20 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "DISCIPLINE_FK_ORGANIZATIONSTRUCTURE" уже существует
Constraint "DISCIPLINE_FK_ORGANIZATIONSTRUCTURE" already exists; SQL statement:
ALTER TABLE Discipline
ADD CONSTRAINT Discipline_fk_OrganizationStructure FOREIGN KEY (OrganizationStructureId) 
  REFERENCES OrganizationStructure (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:25:27 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "DISCIPLINE_FK_SUBJECT" уже существует
Constraint "DISCIPLINE_FK_SUBJECT" already exists; SQL statement:
ALTER TABLE Discipline
ADD CONSTRAINT Discipline_fk_Subject FOREIGN KEY (SubjectId) 
  REFERENCES Subject (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:25:53 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "EDUCATIONPOINTOLD_FK_NAME" уже существует
Constraint "EDUCATIONPOINTOLD_FK_NAME" already exists; SQL statement:
ALTER TABLE EducationPointOld
ADD CONSTRAINT EducationPointOld_fk_Name FOREIGN KEY (NameId) 
  REFERENCES Name (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:26:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "EDUCATIONPOINTOLD_FK_NAME" уже существует
Constraint "EDUCATIONPOINTOLD_FK_NAME" already exists; SQL statement:
ALTER TABLE EducationPointOld
ADD CONSTRAINT EducationPointOld_fk_Name FOREIGN KEY (NameId) 
  REFERENCES Name (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:26:01 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "EDUCATIONPOINTOLD_FK_NAME" уже существует
Constraint "EDUCATIONPOINTOLD_FK_NAME" already exists; SQL statement:
ALTER TABLE EducationPointOld
ADD CONSTRAINT EducationPointOld_fk_Name FOREIGN KEY (NameId) 
  REFERENCES Name (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:29:13 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "EDUCATIONCALENDARITEM_FK_NAME" уже существует
Constraint "EDUCATIONCALENDARITEM_FK_NAME" already exists; SQL statement:
ALTER TABLE EducationCalendarItem
ADD CONSTRAINT EducationCalendarItem_fk_Name FOREIGN KEY (NameId) 
  REFERENCES Name (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:33:47 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "ALTER TABLE EDUCATIONORDER
ADD CONSTRAINT EDUCATIONORDER_FK_ORDER FOREIGN KEY (ORDERID) 
  REFERENCES ORDER[*] (ID) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION "; ожидалось "identifier"
Syntax error in SQL statement "ALTER TABLE EDUCATIONORDER
ADD CONSTRAINT EDUCATIONORDER_FK_ORDER FOREIGN KEY (ORDERID) 
  REFERENCES ORDER[*] (ID) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION "; expected "identifier"; SQL statement:
ALTER TABLE EducationOrder
ADD CONSTRAINT EducationOrder_fk_Order FOREIGN KEY (OrderId) 
  REFERENCES Order (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [42001-196]
2018-03-13 12:34:50 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "ALTER TABLE EDUCATIONORDERBLANK
ADD CONSTRAINT EDUCATIONORDERBLANK_FK_ORDER FOREIGN KEY (ORDERID) 
  REFERENCES ORDER[*] (ID) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION "; ожидалось "identifier"
Syntax error in SQL statement "ALTER TABLE EDUCATIONORDERBLANK
ADD CONSTRAINT EDUCATIONORDERBLANK_FK_ORDER FOREIGN KEY (ORDERID) 
  REFERENCES ORDER[*] (ID) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION "; expected "identifier"; SQL statement:
ALTER TABLE EducationOrderBlank
ADD CONSTRAINT EducationOrderBlank_fk_Order FOREIGN KEY (OrderId) 
  REFERENCES Order (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [42001-196]
2018-03-13 12:36:03 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "PRIVILEGEDCATEGORY_FK_NAME" уже существует
Constraint "PRIVILEGEDCATEGORY_FK_NAME" already exists; SQL statement:
ALTER TABLE PrivilegedCategory
ADD CONSTRAINT PrivilegedCategory_fk_Name FOREIGN KEY (NameId) 
  REFERENCES Name (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:41:29 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "ALTER TABLE ORDERWORK
ADD CONSTRAINT ORDERWORK_FK_ORDER FOREIGN KEY (ORDERID) 
  REFERENCES ORDER[*] (ID) 
  ON UPDATE CASCADE
  ON DELETE CASCADE "; ожидалось "identifier"
Syntax error in SQL statement "ALTER TABLE ORDERWORK
ADD CONSTRAINT ORDERWORK_FK_ORDER FOREIGN KEY (ORDERID) 
  REFERENCES ORDER[*] (ID) 
  ON UPDATE CASCADE
  ON DELETE CASCADE "; expected "identifier"; SQL statement:
ALTER TABLE OrderWork
ADD CONSTRAINT OrderWork_fk_Order FOREIGN KEY (OrderId) 
  REFERENCES Order (Id) 
  ON UPDATE CASCADE
  ON DELETE CASCADE [42001-196]
2018-03-13 12:42:57 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "ORGANIZATIONRELATIONTAG_FK_NAME" уже существует
Constraint "ORGANIZATIONRELATIONTAG_FK_NAME" already exists; SQL statement:
ALTER TABLE OrganizationRelationTag
ADD CONSTRAINT OrganizationRelationTag_fk_Name FOREIGN KEY (NameId) 
  REFERENCES Name (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:44:13 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "SPORTACHIEVEMENT_FK_NAME" уже существует
Constraint "SPORTACHIEVEMENT_FK_NAME" already exists; SQL statement:
ALTER TABLE SportAchievement
ADD CONSTRAINT SportAchievement_fk_Name FOREIGN KEY (NameId) 
  REFERENCES Name (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:44:40 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "STATEAWARD_FK_NAME" уже существует
Constraint "STATEAWARD_FK_NAME" already exists; SQL statement:
ALTER TABLE StateAward
ADD CONSTRAINT StateAward_fk_Name FOREIGN KEY (NameId) 
  REFERENCES Name (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:44:47 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "STATEAWARD_FK_PLACE" уже существует
Constraint "STATEAWARD_FK_PLACE" already exists; SQL statement:
ALTER TABLE StateAward
ADD CONSTRAINT StateAward_fk_Place FOREIGN KEY (CountryPlaceId) 
  REFERENCES Place (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:45:10 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "PHONEOPERATOR_FK_PLACE" уже существует
Constraint "PHONEOPERATOR_FK_PLACE" already exists; SQL statement:
ALTER TABLE PhoneOperator
ADD CONSTRAINT PhoneOperator_fk_Place FOREIGN KEY (PlaceId) 
  REFERENCES Place (Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 12:51:18 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE AUTHORIZATIONIPPMODULE(
  ID INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
  PERSONID INT CONSTRAINT PERSON.[*]ID NOT NULL,
  LOGIN NVARCHAR(50) NOT NULL,
  PASSWORD NVARCHAR(50) NOT NULL,
  ROLE INT NOT NULL) "; ожидалось "PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE AUTHORIZATIONIPPMODULE(
  ID INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
  PERSONID INT CONSTRAINT PERSON.[*]ID NOT NULL,
  LOGIN NVARCHAR(50) NOT NULL,
  PASSWORD NVARCHAR(50) NOT NULL,
  ROLE INT NOT NULL) "; expected "PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE AuthorizationIPPModule(
  Id int PRIMARY KEY auto_increment NOT NULL,
  PersonId int CONSTRAINT Person.Id NOT NULL,
  Login nvarchar(50) NOT NULL,
  Password nvarchar(50) NOT NULL,
  Role int NOT NULL) [42001-196]
2018-03-13 12:57:23 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "ALTER TABLE USERSIPPMODULE
  ADD CONSTRAINT FOREIGN KEY ([[*]PERSONID])
REFERENCES [DBO].[PERSON] ([ID])
  ON UPDATE NO ACTION
  ON DELETE NO ACTION "; ожидалось "identifier"
Syntax error in SQL statement "ALTER TABLE USERSIPPMODULE
  ADD CONSTRAINT FOREIGN KEY ([[*]PERSONID])
REFERENCES [DBO].[PERSON] ([ID])
  ON UPDATE NO ACTION
  ON DELETE NO ACTION "; expected "identifier"; SQL statement:
ALTER TABLE UsersIPPModule
  ADD CONSTRAINT FOREIGN KEY ([PersonId])
REFERENCES [dbo].[Person] ([Id])
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [42001-196]
2018-03-13 13:01:37 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Неизвестный тип данных: "ID"
Unknown data type: "ID"; SQL statement:
CREATE TABLE TeachingLoad(
Id int IDENTITY NOT NULL,
PersonId int NOT NULL,
SemesterId int NOT NULL,
SubjectId int NOT NULL,
EducationYear Id int NOT NULL,
GroupId int NOT NULL,
LecturePlan int NOT NULL,
SeminarPlan int NOT NULL,
LabPlan int NOT NULL,
SSWTPlan int NOT NULL,
ExamPlan int NOT NULL,
PracticePlan int NOT NULL,
DiplomaPlan int NOT NULL,
CourseWorkPlan int NOT NULL,
BoundaryCountedPlan int NOT NULL,
ControlAttendingClassesPlan int NOT NULL,
HistoryKazakhstanPlan int NOT NULL,
StateExamPlan int NOT NULL,
OtherPlan int NOT NULL,
LectureFact int NOT NULL,
SeminarFact int NOT NULL,
LabFact int NOT NULL,
SSWTFact int NOT NULL,
ExamFact int NOT NULL,
PracticeFact int NOT NULL,
DiplomaFact int NOT NULL,
CourseWorkFact int NOT NULL,
BoundaryCountedFact int NOT NULL,
ControlAttendingClassesFact int NOT NULL,
HistoryKazakhstanFact int NOT NULL,
StateExamFact int NOT NULL,
OtherFact int NOT NULL,
isNew bit default (0) not null
) [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseCreateTable(Parser.java:6155)
	at org.h2.command.Parser.parseCreate(Parser.java:4351)
	at org.h2.command.Parser.parsePrepared(Parser.java:366)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 13:02:06 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Неизвестный тип данных: "ID"
Unknown data type: "ID"; SQL statement:
CREATE TABLE TeachingLoad(
Id int IDENTITY NOT NULL,
PersonId int NOT NULL,
SemesterId int NOT NULL,
SubjectId int NOT NULL,
EducationYear Id int NOT NULL,
GroupId int NOT NULL,
LecturePlan int NOT NULL,
SeminarPlan int NOT NULL,
LabPlan int NOT NULL,
SSWTPlan int NOT NULL,
ExamPlan int NOT NULL,
PracticePlan int NOT NULL,
DiplomaPlan int NOT NULL,
CourseWorkPlan int NOT NULL,
BoundaryCountedPlan int NOT NULL,
ControlAttendingClassesPlan int NOT NULL,
HistoryKazakhstanPlan int NOT NULL,
StateExamPlan int NOT NULL,
OtherPlan int NOT NULL,
LectureFact int NOT NULL,
SeminarFact int NOT NULL,
LabFact int NOT NULL,
SSWTFact int NOT NULL,
ExamFact int NOT NULL,
PracticeFact int NOT NULL,
DiplomaFact int NOT NULL,
CourseWorkFact int NOT NULL,
BoundaryCountedFact int NOT NULL,
ControlAttendingClassesFact int NOT NULL,
HistoryKazakhstanFact int NOT NULL,
StateExamFact int NOT NULL,
OtherFact int NOT NULL,
isNew bit default 0 not null
) [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseCreateTable(Parser.java:6155)
	at org.h2.command.Parser.parseCreate(Parser.java:4351)
	at org.h2.command.Parser.parsePrepared(Parser.java:366)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 13:03:35 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Неизвестный тип данных: "ID"
Unknown data type: "ID"; SQL statement:
CREATE TABLE TeachingLoad(
ID int IDENTITY NOT NULL,
PersonId int NOT NULL,
SemesterId int NOT NULL,
SubjectId int NOT NULL,
EducationYear Id int NOT NULL,
GroupId int NOT NULL,
LecturePlan int NOT NULL,
SeminarPlan int NOT NULL,
LabPlan int NOT NULL,
SSWTPlan int NOT NULL,
ExamPlan int NOT NULL,
PracticePlan int NOT NULL,
DiplomaPlan int NOT NULL,
CourseWorkPlan int NOT NULL,
BoundaryCountedPlan int NOT NULL,
ControlAttendingClassesPlan int NOT NULL,
HistoryKazakhstanPlan int NOT NULL,
StateExamPlan int NOT NULL,
OtherPlan int NOT NULL,
LectureFact int NOT NULL,
SeminarFact int NOT NULL,
LabFact int NOT NULL,
SSWTFact int NOT NULL,
ExamFact int NOT NULL,
PracticeFact int NOT NULL,
DiplomaFact int NOT NULL,
CourseWorkFact int NOT NULL,
BoundaryCountedFact int NOT NULL,
ControlAttendingClassesFact int NOT NULL,
HistoryKazakhstanFact int NOT NULL,
StateExamFact int NOT NULL,
OtherFact int NOT NULL,
isNew bit default 0 not null
) [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseCreateTable(Parser.java:6155)
	at org.h2.command.Parser.parseCreate(Parser.java:4351)
	at org.h2.command.Parser.parsePrepared(Parser.java:366)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 13:03:56 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "CREATE TABLE TEACHINGLOAD(
ID INT PRIMARY KEY AUTO_INCREMENTNOT[*] NULL,
PERSONID INT NOT NULL,
SEMESTERID INT NOT NULL,
SUBJECTID INT NOT NULL,
EDUCATIONYEAR ID INT NOT NULL,
GROUPID INT NOT NULL,
LECTUREPLAN INT NOT NULL,
SEMINARPLAN INT NOT NULL,
LABPLAN INT NOT NULL,
SSWTPLAN INT NOT NULL,
EXAMPLAN INT NOT NULL,
PRACTICEPLAN INT NOT NULL,
DIPLOMAPLAN INT NOT NULL,
COURSEWORKPLAN INT NOT NULL,
BOUNDARYCOUNTEDPLAN INT NOT NULL,
CONTROLATTENDINGCLASSESPLAN INT NOT NULL,
HISTORYKAZAKHSTANPLAN INT NOT NULL,
STATEEXAMPLAN INT NOT NULL,
OTHERPLAN INT NOT NULL,
LECTUREFACT INT NOT NULL,
SEMINARFACT INT NOT NULL,
LABFACT INT NOT NULL,
SSWTFACT INT NOT NULL,
EXAMFACT INT NOT NULL,
PRACTICEFACT INT NOT NULL,
DIPLOMAFACT INT NOT NULL,
COURSEWORKFACT INT NOT NULL,
BOUNDARYCOUNTEDFACT INT NOT NULL,
CONTROLATTENDINGCLASSESFACT INT NOT NULL,
HISTORYKAZAKHSTANFACT INT NOT NULL,
STATEEXAMFACT INT NOT NULL,
OTHERFACT INT NOT NULL,
ISNEW BIT DEFAULT 0 NOT NULL
) "; ожидалось "HASH, AUTO_INCREMENT, NOT, NULL, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE TEACHINGLOAD(
ID INT PRIMARY KEY AUTO_INCREMENTNOT[*] NULL,
PERSONID INT NOT NULL,
SEMESTERID INT NOT NULL,
SUBJECTID INT NOT NULL,
EDUCATIONYEAR ID INT NOT NULL,
GROUPID INT NOT NULL,
LECTUREPLAN INT NOT NULL,
SEMINARPLAN INT NOT NULL,
LABPLAN INT NOT NULL,
SSWTPLAN INT NOT NULL,
EXAMPLAN INT NOT NULL,
PRACTICEPLAN INT NOT NULL,
DIPLOMAPLAN INT NOT NULL,
COURSEWORKPLAN INT NOT NULL,
BOUNDARYCOUNTEDPLAN INT NOT NULL,
CONTROLATTENDINGCLASSESPLAN INT NOT NULL,
HISTORYKAZAKHSTANPLAN INT NOT NULL,
STATEEXAMPLAN INT NOT NULL,
OTHERPLAN INT NOT NULL,
LECTUREFACT INT NOT NULL,
SEMINARFACT INT NOT NULL,
LABFACT INT NOT NULL,
SSWTFACT INT NOT NULL,
EXAMFACT INT NOT NULL,
PRACTICEFACT INT NOT NULL,
DIPLOMAFACT INT NOT NULL,
COURSEWORKFACT INT NOT NULL,
BOUNDARYCOUNTEDFACT INT NOT NULL,
CONTROLATTENDINGCLASSESFACT INT NOT NULL,
HISTORYKAZAKHSTANFACT INT NOT NULL,
STATEEXAMFACT INT NOT NULL,
OTHERFACT INT NOT NULL,
ISNEW BIT DEFAULT 0 NOT NULL
) "; expected "HASH, AUTO_INCREMENT, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE TeachingLoad(
Id int PRIMARY KEY AUTO_INCREMENTNOT NULL,
PersonId int NOT NULL,
SemesterId int NOT NULL,
SubjectId int NOT NULL,
EducationYear Id int NOT NULL,
GroupId int NOT NULL,
LecturePlan int NOT NULL,
SeminarPlan int NOT NULL,
LabPlan int NOT NULL,
SSWTPlan int NOT NULL,
ExamPlan int NOT NULL,
PracticePlan int NOT NULL,
DiplomaPlan int NOT NULL,
CourseWorkPlan int NOT NULL,
BoundaryCountedPlan int NOT NULL,
ControlAttendingClassesPlan int NOT NULL,
HistoryKazakhstanPlan int NOT NULL,
StateExamPlan int NOT NULL,
OtherPlan int NOT NULL,
LectureFact int NOT NULL,
SeminarFact int NOT NULL,
LabFact int NOT NULL,
SSWTFact int NOT NULL,
ExamFact int NOT NULL,
PracticeFact int NOT NULL,
DiplomaFact int NOT NULL,
CourseWorkFact int NOT NULL,
BoundaryCountedFact int NOT NULL,
ControlAttendingClassesFact int NOT NULL,
HistoryKazakhstanFact int NOT NULL,
StateExamFact int NOT NULL,
OtherFact int NOT NULL,
isNew bit default 0 not null
) [42001-196]
2018-03-13 13:04:18 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Неизвестный тип данных: "ID"
Unknown data type: "ID"; SQL statement:
CREATE TABLE TeachingLoad(
Id int PRIMARY KEY AUTO_INCREMENT NOT NULL,
PersonId int NOT NULL,
SemesterId int NOT NULL,
SubjectId int NOT NULL,
EducationYear Id int NOT NULL,
GroupId int NOT NULL,
LecturePlan int NOT NULL,
SeminarPlan int NOT NULL,
LabPlan int NOT NULL,
SSWTPlan int NOT NULL,
ExamPlan int NOT NULL,
PracticePlan int NOT NULL,
DiplomaPlan int NOT NULL,
CourseWorkPlan int NOT NULL,
BoundaryCountedPlan int NOT NULL,
ControlAttendingClassesPlan int NOT NULL,
HistoryKazakhstanPlan int NOT NULL,
StateExamPlan int NOT NULL,
OtherPlan int NOT NULL,
LectureFact int NOT NULL,
SeminarFact int NOT NULL,
LabFact int NOT NULL,
SSWTFact int NOT NULL,
ExamFact int NOT NULL,
PracticeFact int NOT NULL,
DiplomaFact int NOT NULL,
CourseWorkFact int NOT NULL,
BoundaryCountedFact int NOT NULL,
ControlAttendingClassesFact int NOT NULL,
HistoryKazakhstanFact int NOT NULL,
StateExamFact int NOT NULL,
OtherFact int NOT NULL,
isNew bit default 0 not null
) [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseCreateTable(Parser.java:6155)
	at org.h2.command.Parser.parseCreate(Parser.java:4351)
	at org.h2.command.Parser.parsePrepared(Parser.java:366)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 13:04:47 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Неизвестный тип данных: "ID"
Unknown data type: "ID"; SQL statement:
CREATE TABLE TeachingLoad(
  Id int PRIMARY KEY AUTO_INCREMENT NOT NULL,
PersonId int NOT NULL,
SemesterId int NOT NULL,
SubjectId int NOT NULL,
EducationYear Id int NOT NULL,
GroupId int NOT NULL,
LecturePlan int NOT NULL,
SeminarPlan int NOT NULL,
LabPlan int NOT NULL,
SSWTPlan int NOT NULL,
ExamPlan int NOT NULL,
PracticePlan int NOT NULL,
DiplomaPlan int NOT NULL,
CourseWorkPlan int NOT NULL,
BoundaryCountedPlan int NOT NULL,
ControlAttendingClassesPlan int NOT NULL,
HistoryKazakhstanPlan int NOT NULL,
StateExamPlan int NOT NULL,
OtherPlan int NOT NULL,
LectureFact int NOT NULL,
SeminarFact int NOT NULL,
LabFact int NOT NULL,
SSWTFact int NOT NULL,
ExamFact int NOT NULL,
PracticeFact int NOT NULL,
DiplomaFact int NOT NULL,
CourseWorkFact int NOT NULL,
BoundaryCountedFact int NOT NULL,
ControlAttendingClassesFact int NOT NULL,
HistoryKazakhstanFact int NOT NULL,
StateExamFact int NOT NULL,
OtherFact int NOT NULL,
isNew bit default 0 not null
) [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseCreateTable(Parser.java:6155)
	at org.h2.command.Parser.parseCreate(Parser.java:4351)
	at org.h2.command.Parser.parsePrepared(Parser.java:366)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 13:07:34 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Неизвестный тип данных: "ID"
Unknown data type: "ID"; SQL statement:
CREATE TABLE TeachingLoad(
  Id int AUTO_INCREMENT NOT NULL,
PersonId int NOT NULL,
SemesterId int NOT NULL,
SubjectId int NOT NULL,
EducationYear Id int NOT NULL,
GroupId int NOT NULL,
LecturePlan int NOT NULL,
SeminarPlan int NOT NULL,
LabPlan int NOT NULL,
SSWTPlan int NOT NULL,
ExamPlan int NOT NULL,
PracticePlan int NOT NULL,
DiplomaPlan int NOT NULL,
CourseWorkPlan int NOT NULL,
BoundaryCountedPlan int NOT NULL,
ControlAttendingClassesPlan int NOT NULL,
HistoryKazakhstanPlan int NOT NULL,
StateExamPlan int NOT NULL,
OtherPlan int NOT NULL,
LectureFact int NOT NULL,
SeminarFact int NOT NULL,
LabFact int NOT NULL,
SSWTFact int NOT NULL,
ExamFact int NOT NULL,
PracticeFact int NOT NULL,
DiplomaFact int NOT NULL,
CourseWorkFact int NOT NULL,
BoundaryCountedFact int NOT NULL,
ControlAttendingClassesFact int NOT NULL,
HistoryKazakhstanFact int NOT NULL,
StateExamFact int NOT NULL,
OtherFact int NOT NULL,
isNew bit default 0 not null
) [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseCreateTable(Parser.java:6155)
	at org.h2.command.Parser.parseCreate(Parser.java:4351)
	at org.h2.command.Parser.parsePrepared(Parser.java:366)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 13:07:44 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Неизвестный тип данных: "ID"
Unknown data type: "ID"; SQL statement:
CREATE TABLE TeachingLoad (
  Id int PRIMARY KEY AUTO_INCREMENT NOT NULL,
PersonId int NOT NULL,
SemesterId int NOT NULL,
SubjectId int NOT NULL,
EducationYear Id int NOT NULL,
GroupId int NOT NULL,
LecturePlan int NOT NULL,
SeminarPlan int NOT NULL,
LabPlan int NOT NULL,
SSWTPlan int NOT NULL,
ExamPlan int NOT NULL,
PracticePlan int NOT NULL,
DiplomaPlan int NOT NULL,
CourseWorkPlan int NOT NULL,
BoundaryCountedPlan int NOT NULL,
ControlAttendingClassesPlan int NOT NULL,
HistoryKazakhstanPlan int NOT NULL,
StateExamPlan int NOT NULL,
OtherPlan int NOT NULL,
LectureFact int NOT NULL,
SeminarFact int NOT NULL,
LabFact int NOT NULL,
SSWTFact int NOT NULL,
ExamFact int NOT NULL,
PracticeFact int NOT NULL,
DiplomaFact int NOT NULL,
CourseWorkFact int NOT NULL,
BoundaryCountedFact int NOT NULL,
ControlAttendingClassesFact int NOT NULL,
HistoryKazakhstanFact int NOT NULL,
StateExamFact int NOT NULL,
OtherFact int NOT NULL,
isNew bit default 0 not null
) [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseCreateTable(Parser.java:6155)
	at org.h2.command.Parser.parseCreate(Parser.java:4351)
	at org.h2.command.Parser.parsePrepared(Parser.java:366)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 13:09:01 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Неизвестный тип данных: "ID"
Unknown data type: "ID"; SQL statement:
CREATE TABLE TeachingLoad (
  Id bigint PRIMARY KEY AUTO_INCREMENT NOT NULL,
PersonId int NOT NULL,
SemesterId int NOT NULL,
SubjectId int NOT NULL,
EducationYear Id int NOT NULL,
GroupId int NOT NULL,
LecturePlan int NOT NULL,
SeminarPlan int NOT NULL,
LabPlan int NOT NULL,
SSWTPlan int NOT NULL,
ExamPlan int NOT NULL,
PracticePlan int NOT NULL,
DiplomaPlan int NOT NULL,
CourseWorkPlan int NOT NULL,
BoundaryCountedPlan int NOT NULL,
ControlAttendingClassesPlan int NOT NULL,
HistoryKazakhstanPlan int NOT NULL,
StateExamPlan int NOT NULL,
OtherPlan int NOT NULL,
LectureFact int NOT NULL,
SeminarFact int NOT NULL,
LabFact int NOT NULL,
SSWTFact int NOT NULL,
ExamFact int NOT NULL,
PracticeFact int NOT NULL,
DiplomaFact int NOT NULL,
CourseWorkFact int NOT NULL,
BoundaryCountedFact int NOT NULL,
ControlAttendingClassesFact int NOT NULL,
HistoryKazakhstanFact int NOT NULL,
StateExamFact int NOT NULL,
OtherFact int NOT NULL,
IsNew bit default 0 not null
) [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseCreateTable(Parser.java:6155)
	at org.h2.command.Parser.parseCreate(Parser.java:4351)
	at org.h2.command.Parser.parsePrepared(Parser.java:366)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 13:09:16 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Неизвестный тип данных: "ID"
Unknown data type: "ID"; SQL statement:
CREATE TABLE TeachingLoad (
  Id bigint PRIMARY KEY AUTO_INCREMENT,
PersonId int NOT NULL,
SemesterId int NOT NULL,
SubjectId int NOT NULL,
EducationYear Id int NOT NULL,
GroupId int NOT NULL,
LecturePlan int NOT NULL,
SeminarPlan int NOT NULL,
LabPlan int NOT NULL,
SSWTPlan int NOT NULL,
ExamPlan int NOT NULL,
PracticePlan int NOT NULL,
DiplomaPlan int NOT NULL,
CourseWorkPlan int NOT NULL,
BoundaryCountedPlan int NOT NULL,
ControlAttendingClassesPlan int NOT NULL,
HistoryKazakhstanPlan int NOT NULL,
StateExamPlan int NOT NULL,
OtherPlan int NOT NULL,
LectureFact int NOT NULL,
SeminarFact int NOT NULL,
LabFact int NOT NULL,
SSWTFact int NOT NULL,
ExamFact int NOT NULL,
PracticeFact int NOT NULL,
DiplomaFact int NOT NULL,
CourseWorkFact int NOT NULL,
BoundaryCountedFact int NOT NULL,
ControlAttendingClassesFact int NOT NULL,
HistoryKazakhstanFact int NOT NULL,
StateExamFact int NOT NULL,
OtherFact int NOT NULL,
IsNew bit default 0 not null
) [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseCreateTable(Parser.java:6155)
	at org.h2.command.Parser.parseCreate(Parser.java:4351)
	at org.h2.command.Parser.parsePrepared(Parser.java:366)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 13:39:40 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Таблица "ACADEMICYEAR" не найдена
Table "ACADEMICYEAR" not found; SQL statement:
ALTER TABLE TeachingLoad
ADD CONSTRAINT TeachingLoad_fk_AcademicYear FOREIGN KEY (SubjectId) 
  REFERENCES AcademicYear(Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [42102-196]
2018-03-13 13:41:49 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "ALTER TABLE TEACHINGLOAD
DELETE[*] ACADEMICYEARID


ADD CONSTRAINT TEACHINGLOAD_FK_ACADEMICYEAR FOREIGN KEY (SUBJECTID) 
  REFERENCES ACADEMICYEAR(ID) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION "; ожидалось "., ADD, SET, RENAME, DROP, CHANGE, MODIFY, ALTER"
Syntax error in SQL statement "ALTER TABLE TEACHINGLOAD
DELETE[*] ACADEMICYEARID


ADD CONSTRAINT TEACHINGLOAD_FK_ACADEMICYEAR FOREIGN KEY (SUBJECTID) 
  REFERENCES ACADEMICYEAR(ID) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION "; expected "., ADD, SET, RENAME, DROP, CHANGE, MODIFY, ALTER"; SQL statement:
ALTER TABLE TeachingLoad
delete AcademicYearId


ADD CONSTRAINT TeachingLoad_fk_AcademicYear FOREIGN KEY (SubjectId) 
  REFERENCES AcademicYear(Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [42001-196]
2018-03-13 13:41:58 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "ALTER TABLE TEACHINGLOAD
DELETE[*] ACADEMICYEARID "; ожидалось "., ADD, SET, RENAME, DROP, CHANGE, MODIFY, ALTER"
Syntax error in SQL statement "ALTER TABLE TEACHINGLOAD
DELETE[*] ACADEMICYEARID "; expected "., ADD, SET, RENAME, DROP, CHANGE, MODIFY, ALTER"; SQL statement:
ALTER TABLE TeachingLoad
delete AcademicYearId [42001-196]
2018-03-13 15:31:46 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Ограничение "TEACHINGLOAD_FK_SUBJECT" уже существует
Constraint "TEACHINGLOAD_FK_SUBJECT" already exists; SQL statement:
ALTER TABLE TeachingLoad
ADD CONSTRAINT TeachingLoad_fk_Subject FOREIGN KEY (SubjectId ) 
  REFERENCES Subject(Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [90045-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:77)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-13 15:32:24 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "ALTER TABLE TEACHINGLOAD
ADD CONSTRAINT TEACHINGLOAD_FK_GROUP FOREIGN KEY (GROUPID ) 
  REFERENCES GROUP[*](ID) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION "; ожидалось "identifier"
Syntax error in SQL statement "ALTER TABLE TEACHINGLOAD
ADD CONSTRAINT TEACHINGLOAD_FK_GROUP FOREIGN KEY (GROUPID ) 
  REFERENCES GROUP[*](ID) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION "; expected "identifier"; SQL statement:
ALTER TABLE TeachingLoad
ADD CONSTRAINT TeachingLoad_fk_Group FOREIGN KEY (GroupId ) 
  REFERENCES Group(Id) 
  ON UPDATE NO ACTION
  ON DELETE NO ACTION [42001-196]
2018-03-13 19:21:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
SELECT id,role FROM Users WHERE login=? AND password=? [42102-195]
2018-03-13 19:24:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
SELECT id,role FROM Users WHERE login=? AND password=? [42102-195]
2018-03-13 20:18:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
SELECT id,role FROM Users WHERE login=? AND password=? [42102-195]
2018-03-14 10:35:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Столбец "TIMENORMS" не найден
Column "TIMENORMS" not found; SQL statement:
SELECT TIMENORMS [42122-195]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Select.prepare(Select.java:839)
	at org.h2.command.Parser.prepareCommand(Parser.java:262)
	at org.h2.engine.Session.prepareLocal(Session.java:573)
	at org.h2.engine.Session.prepareCommand(Session.java:514)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:165)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-14 10:44:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "UPDATE TIMENORMS SET CODE='1102' WHERE ID=2
SELECT[*] * FROM  TIMENORMS "
Syntax error in SQL statement "UPDATE TIMENORMS SET CODE='1102' WHERE ID=2
SELECT[*] * FROM  TIMENORMS "; SQL statement:
UPDATE TimeNorms SET code='1102' where id=2
SELECT * from  TIMENORMS [42000-195]
2018-03-14 13:36:48 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Неверное количество столбцов
Column count does not match; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('1114', 'Подготовка к занятиям: Кейс', 0) [21002-195]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepareCommand(Parser.java:262)
	at org.h2.engine.Session.prepareLocal(Session.java:573)
	at org.h2.engine.Session.prepareCommand(Session.java:514)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:165)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01101', 'Подготовка к занятиям: К лекциям по новому (для преподавателя) курсу', 2,1) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01102', 'Подготовка к занятиям: К лекциям по читаемому курсу', 0,1) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01103', 'Подготовка к занятиям: К практическим (семинарским) занятиям', 0,1) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01104', 'Подготовка к занятиям: К лабораторным работам', 0,1) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01105', 'Подготовка к занятиям: К выездным занятиям, туристским походам', 0,1) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01106', 'Подготовка к занятиям: К  практическим  занятиям, лабораторным по  новому для преподавателя курсу', 0,1) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01107', 'Подготовка к занятиям: СРСП, СРМП, СРДП', 0,1) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01108', 'Подготовка к занятиям: Открытому занятию', 0,1) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01109', 'Подготовка к занятиям: Подготовка и запись видеолекций', 0,1) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01110', 'Подготовка к занятиям: Подготовка и проведение показательных занятий', 0,1) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01111', 'Подготовка к занятиям: Подготовка и проведение онлайн-лекций', 0,1) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01112', 'Подготовка к занятиям: Ролевая игра, деловая игра', 0,1) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01113', 'Подготовка к занятиям: Виртуальные лабораторные работы', 0,1) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01114', 'Подготовка к занятиям: Кейс', 0) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01115', 'Подготовка к занятиям: Занятия с участием работодателей', 0,1) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01201', 'Разработка текстов лекций по новому курсу ', 0,2) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01301', 'Разработка заданий: На дипломную работу', 0,3) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01401', 'Написание методических указаний: Для дипломной работы', 0,4) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01402', 'Написание методических указаний: Для курсовой работы', 0,4) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01403', 'Написание методических указаний: Для практических, семинарских занятий', 0,4) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01404', 'Написание методических указаний: Для лабораторных работ', 0,4) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01405', 'Написание методических указаний: Сборников упражнений, задач', 0,4) [42102-195]
2018-03-14 13:48:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TIMENORMS" не найдена
Table "TIMENORMS" not found; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01406', 'Написание методических указаний: Для прохождения учебной (производственной) практики', 0,4) [42102-195]
2018-03-14 13:48:14 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Неверное количество столбцов
Column count does not match; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01114', 'Подготовка к занятиям: Кейс', 0) [21002-195]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepareCommand(Parser.java:262)
	at org.h2.engine.Session.prepareLocal(Session.java:573)
	at org.h2.engine.Session.prepareCommand(Session.java:514)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:165)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-14 15:30:50 jdbc[5]: null
org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-195]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:185)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:127)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:100)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.login(WebApp.java:947)
	at org.h2.server.web.WebApp.process(WebApp.java:210)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:169)
	at org.h2.server.web.WebThread.process(WebThread.java:133)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-195]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 14 more
2018-03-14 15:31:04 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "TIMENORMS[*]"; ожидалось "TRUNCATE, {"
Syntax error in SQL statement "TIMENORMS[*]"; expected "TRUNCATE, {"; SQL statement:
TIMENORMS [42001-195]
2018-03-14 15:42:12 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Значение слишком длинное для поля "WORKTYPE NVARCHAR(100) NOT NULL": "STRINGDECODE('\u0420\u0430\u0437\u0440\u0430\u0431\u043e\u0442\u043a\u0430 \u0438 \u0432\u043d\u0435\u0434\u0440\u0435\u043d\u04... (147)"
Value too long for column "WORKTYPE NVARCHAR(100) NOT NULL": "STRINGDECODE('\u0420\u0430\u0437\u0440\u0430\u0431\u043e\u0442\u043a\u0430 \u0438 \u0432\u043d\u0435\u0434\u0440\u0435\u043d\u04... (147)"; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('10011', 'Разработка и внедрение: Раздаточных информационных средств (лекционных демонстраций, схем, фильмов, имитаторов, электронных   презентаций, слайдов)', 0,1) [22001-195]
2018-03-14 15:43:42 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Неверное количество столбцов
Column count does not match; SQL statement:
INSERT INTO TimeNorms(Code,  WorkType,  Hours, IdTypeWork) 
VALUES ('01114', 'Подготовка к занятиям: Кейс', 0) [21002-195]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepareCommand(Parser.java:262)
	at org.h2.engine.Session.prepareLocal(Session.java:573)
	at org.h2.engine.Session.prepareCommand(Session.java:514)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:165)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
2018-03-14 18:38:49 jdbc[6]: null
org.h2.message.DbException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-195]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:185)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:127)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:100)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.login(WebApp.java:947)
	at org.h2.server.web.WebApp.process(WebApp.java:210)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:169)
	at org.h2.server.web.WebThread.process(WebThread.java:133)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Незакрытое приложением соединение уничтожено сборщиком мусора
The connection was not closed by the application and is garbage collected [90018-195]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 14 more
